packages/s/streamlink/package.yml

```diff
@@ -23,6 +22,7 @@ builddeps  :
     - python-trio-websocket
     - python-typing-extensions
     - python-websocket-client
+    - python3-pycryptodome
 checkdeps  :
     - python-freezegun
     - python-pytest-asyncio
```

Comment:
The `pytest-asyncio` dev-requirement has been removed in 6.11.0:
https://streamlink.github.io/changelog.html#streamlink-6-11-0-2024-10-01

Reply:
Thanks, all the tests are passing now with the upstream changes